ios/Nodes/REATransformNode.m

```diff
@@ -20,7 +20,7 @@ - (id)evaluate
   NSMutableArray<NSDictionary *> *transform = [NSMutableArray arrayWithCapacity:_transformConfigs.count];
   for (NSDictionary *transformConfig in _transformConfigs) {
     NSString *property = transformConfig[@"property"];
-    REANodeID nodeID = transformConfig[@"nodeID"];
+    REANodeID nodeID = [RCTConvert NSNumber:transformConfig[@"nodeID"]];
```

Comment:
why. It worked before this change. If it's something you consider important, but not related, open another PR.